README.md

```diff
@@ -61,6 +61,16 @@ In order to run the tests for the different test suites and decoders, a
 `resources` directory containing all the input files for each test suite needs
 to be collected first.
 
+## How to get started
+
+- Clone the repository: gh repo clone fluendo/fluster 
```

Comment:
```suggestion
- Clone the repository: `git clone https://github.com/fluendo/fluster.git` 
```

Better to use `git` than `gh`. It's more generic and nor only related with GitHub.